### DISCUSSION TITLE
处理表单多对一的问题

### DISCUSSION THREAD

**Discussion by lck123357** [alibaba/formily#1981] [Q&A 2.x] [answered]
假设：表单内一个字段的选项列表，需要依赖其他两个以上字段的值进行条件获取

**Comment by janryWang** [comment, marked as answer]
1.x不支持依赖型联动

**Reply by lck123357** [reply]
不好意思 这边用的是2.x的，刚才写错了，麻烦您指导一下

**Comment by janryWang** [comment]
文档都有的，不要问文档已有的。。。https://v2.formilyjs.org/zh-CN/guide/advanced/linkages#schemareactions-%E7%94%A8%E4%BE%8B-9

**Reply by lck123357** [reply]
感谢
